packages/expo-random/src/__tests__/Random-test.ts

```diff
@@ -0,0 +1,21 @@
+import { unmockProperty } from 'jest-expo';
+
+import ExpoRandom from '../ExpoRandom';
+import * as Random from '../Random';
+
+it(`invokes native method correctly`, async () => {
+  ExpoRandom.getRandomBytesAsync.mockImplementation(async () => '');
+  const value = await Random.getRandomBytesAsync(0);
+
+  expect(value instanceof Uint8Array).toBe(true);
+  expect(ExpoRandom.getRandomBytesAsync).toHaveBeenLastCalledWith(0);
+
+  unmockProperty(ExpoRandom, 'getRandomBytesAsync');
```

Comment:
This does nothing since we use `mockImplementation` instead of `mockProperty` - can remove the unmockProperty calls in this test.

To clear the mock implementation there are three options:
- Call `ExpoRandom.getRandomBytesAsync.mockReset()`
- Call `jest.resetAllMocks()`
- Use `ExpoRandom.getRandomBytesAsync.mockImplementationOnce(...)`

In this case, `getRandomBytesAsync` is the only mocked function so these all do the same. I'd go with `mockImplementationOnce` or `jest.resetAllMocks` as the simpler two of the three options.